sources/imdsclient/src/lib.rs

```diff
@@ -19,9 +19,8 @@ The result is returned as a `String` _(ex. m5.large)_.
 
 use std::sync::RwLock;
 
-use http::StatusCode;
 use log::{debug, info, trace, warn};
-use reqwest::Client;
+use reqwest::{Client, StatusCode};
```

Comment:
It'd be good to add a short explanation of why this is changing (rather than just "what" is changing).